data/core/keymap.lua

```diff
@@ -70,6 +103,58 @@ local function key_to_stroke(key)
 end
 
 
+---Returns the value at `tbl[keys[1]]...[keys[#keys]]`
+---@param tbl table<any, any> Base table
+---@param keys any[] Keys
+---@return any? `tbl[keys[1]]...[keys[#keys]]`
+local function get_nested(tbl, keys)
+  local r, i = tbl, 1
+  while r and i < #keys + 1 do
+    r = type(r) == "table" and r[keys[i]] or nil
+    i = i + 1
+  end
+  return r
+end
```

Comment:
Let's avoid using explicit indexes
```suggestion
---Returns the value at `tbl[keys[1]]...[keys[#keys]]`
---@param tbl table<any, any> Base table
---@param keys any[] Keys
---@return any? `tbl[keys[1]]...[keys[#keys]]`
local function get_nested(tbl, keys)
  local r = tbl
  for _, key in ipairs(keys) do
    r = type(r) == "table" and r[key] or nil
    if not r then break end
  end
  return r
end
```